### ISSUE TITLE
Add a 1.0 release to JitPack

### ISSUE THREAD

**Issue by Wavesonics** [KeepSafe/ReLinker#1]
https://jitpack.io/#KeepSafe/ReLinker reports that no release has been tagged:
Repository not found or requires authentication
No releases found. Use the short commit id or `-SNAPSHOT`

Attempting to include the library using the provided dependency line:
compile 'com.github.KeepSafe:ReLinker:1.0'

Reports that it can not be found.

**Comment by xiphirx** [comment]
Hi, I'm not sure why jitpack is not showing any tagged releases, as there is a tagged release: https://github.com/KeepSafe/ReLinker/releases/tag/1.0 

I went ahead and tried to include the library into an empty project and it recognized and downloaded the dependencies without issue... Did you include the jitpack repository?

```
repositories { 
        jcenter()
        maven { url "https://jitpack.io" }
   }
```

**Comment by xiphirx** [comment]
jitpack had an issue where they thought the repo was private. It should show the release on their page now. Closing this, re-open if you still cannot get the dependency.

**Comment by Wavesonics** [comment]
Great, all working now, thanks!

And awesome work tracking this issue down! How in the world has this bug persisted in the PackageManger this long?

Did anyone at Google acknowledge it? Maybe indicate that they would fix it in the OS at some point?

**Comment by xiphirx** [comment]
At the end of our blog post there is a comment that was extracted from Chromium's source code. The comment says that the bug was internally found, later it was fixed. It apparently shouldn't be an issue anymore for newer APIs (18+ I believe?). We submitted a bug report years ago but Google didn't act on it, they just marked it obsolete a few months ago instead.

Hopefully this library resolves the issue for you!
